byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Block.hs

```diff
@@ -122,11 +121,10 @@ bHeaderSize = fromIntegral . abstractSize costs
 
 -- | Computes the hash of a header.
 hashHeader :: BlockHeader -> Hash
-hashHeader bh = hashlazy . pack $
-  show (bh ^. bhPrevHash) ++
-  show (bh ^. bhSlot    ) ++
-  show (bh ^. bhIssuer  )
--- TODO: we might want to serialize this properly, without using show...
+hashHeader bh =
+  (bh ^. bhPrevHash)
+    `hashWithSalt` (bh ^. bhSlot)
+    `hashWithSalt` (bh ^. bhIssuer)
```

Comment:
This could be just `hash (_, _, _)` rather than chaining with `hashWithSalt`.

Reply:
Why do we not hash the whole content of the header, covering all fields?

Reply:
Yeah, but this is `hashable`s recommended way of doing things :man_shrugging:

Reply:
Changed it to triple instead ;)